andela/travel_tool_front
# #159386918 Run test coverage

- Run codeclimate to get test coverage after running circleci

[delivers: #159386918]

#### What does this PR do?
- It runs test coverage after running on circleci.

#### Description of Task to be completed?
- Run codeclimate test coverage after passing the tests from circleci

#### How should this be manually tested?
- Visit this link: https://circleci.com/gh/andela/travel_tool_front/61, at the end you will see a message:
`Test report uploaded successfully to Code Climate`

#### What are the relevant pivotal tracker stories?
- #159386918
